adapters/conversant/conversant.go

```diff
@@ -74,8 +74,14 @@ func (c ConversantAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *a
 func parseCnvrParams(imp *openrtb2.Imp, cnvrExt openrtb_ext.ExtImpConversant) {
 	imp.DisplayManager = "prebid-s2s"
 	imp.DisplayManagerVer = "2.0.0"
-	imp.BidFloor = cnvrExt.BidFloor
-	imp.TagID = cnvrExt.TagID
+
+	if imp.BidFloor <= 0 && cnvrExt.BidFloor > 0 {
+		imp.BidFloor = cnvrExt.BidFloor
+	}
```

Comment:
In case the `cnvrExt.BidFloor` is zero, and `imp.BidFloor` is negative, `imp.BidFloor` will continue being negative. Do we mind this? In your context, does it make sense to default `imp.BidFloor` to a non-negative value?
```
74   func parseCnvrParams(imp *openrtb2.Imp, cnvrExt openrtb_ext.ExtImpConversant) {
75       imp.DisplayManager = "prebid-s2s"
76       imp.DisplayManagerVer = "2.0.0"
77  
78       if imp.BidFloor <= 0 && cnvrExt.BidFloor > 0 {
79           imp.BidFloor = cnvrExt.BidFloor
80       }
   +      
   +     if imp.BidFloor < 0 {
   +         // set to default value
   +         imp.BidFloor = defaultBidFloorValue
   +     }
81  
82       if len(cnvrExt.TagID) > 0 {
83           imp.TagID = cnvrExt.TagID
84       }
85  
86       // Take care not to override the global secure flag
87       if (imp.Secure == nil || *imp.Secure == 0) && cnvrExt.Secure != nil {
88           imp.Secure = cnvrExt.Secure
89       }
adapters/conversant/conversant.go
```

Reply:
We treat zero and negative bid floors the same so it won't make a difference.